### PR TITLE
Add recipe for lintegrate

### PR DESCRIPTION
**Pull request by mattpitkin**
Hi @conda-forge/help-python-c,

I have a package that I think is ready for review. The package builds locally for me using the instructions give [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#staging-test-locally).

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/lintegrate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - numpy
+    - python
+
+test:
+  imports:
+    - lintegrate
+
+about:
+  home: http://github.com/mattpitkin/lintegrate
+  license: GPLv3
```
Comment (chrisburr) [inline recipes/lintegrate/meta.yaml]: SPDX identifiers are preferred:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/lintegrate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  license_family: GPL
+  license_file: LICENSE.txt
```
Comment (chrisburr) [inline recipes/lintegrate/meta.yaml]: Looking at the manifest it looks like the licence is already in the tarball
```suggestion
  license_file: LICENSE
```

File: recipes/lintegrate/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (chrisburr) [inline recipes/lintegrate/LICENSE.txt]: If the licence is in the tarball this can be deleted

Comment (mattpitkin) [reply recipes/lintegrate/LICENSE.txt]: Thanks, I've deleted the license file now.

**Pull request merged**